src/css/_navigation.scss

```diff
@@ -83,7 +83,16 @@
   font-size: 1em;
 }
 
-@media (max-width: 767.98px) {
+@media (max-width: 575.98px) {
```

Comment:
Isn't this reverting what was done in the other PR?

Reply:
@Moeez905 should there simply be no changes to `_navigation.scss` in this PR?

Reply:
> I've totally lost track of the proper changes here

Let me know if I can assist

Reply:
> @Moeez905 should there simply be no changes to `_navigation.scss` in this PR?

Yes! there must be no change in navigation.css as its "footer's" PR